### PR TITLE
Minor: typos, grammar

### PR DESCRIPTION
**Pull request by ftobia**
Hello, I saw a minor typo or two, and while I was fixing them I made some grammar tweaks as well. I tried wherever possible to keep the previous intent. Some of them I think can go either way, and I am happy to undo any of these changes. Please let me know.

### CODE REVIEW

**Comment by ftobia** [comment]
@meeseeksdev tag documentation

File: docs/source/interactive/autoawait.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ By default IPython will assume integration with Python's provided
 :mod:`asyncio`, but integration with other libraries is provided. In particular
 we provide experimental integration with the ``curio`` and ``trio`` library.
 
-You can switch current integration by using the
+You can switch the current integration by using the
```
Comment (ftobia) [inline docs/source/interactive/autoawait.rst:76]: This one I am 50/50 on.

File: docs/source/interactive/autoawait.rst [hunk 2]
```diff
@@ -131,23 +131,23 @@ a loop to run. By default IPython will use a fake coroutine runner which should
 allow ``IPython.embed()`` to be nested. Though this will prevent usage of the
 :magic:`%autoawait` feature when using IPython embed. 
 
-You can set explicitly a coroutine runner for ``embed()`` if you desire to run
-asynchronous code, the exact behavior is though undefined.
+You can set a coroutine runner explicitly for ``embed()`` if you want to run
+asynchronous code, though the exact behavior is undefined.
```
Comment (ftobia) [inline docs/source/interactive/autoawait.rst:135]: I rephrased for flow but perhaps this is gratuitous. I am happy to change it back.

**Comment by Carreau** [comment]
Thanks !

File: docs/source/interactive/autoawait.rst (hunk 2 above)
Comment (Carreau) [reply docs/source/interactive/autoawait.rst:135]: Thanks ! I'm not a good english writer ! Much aprreciated.

**Pull request merged**